snap/snapcraft.yaml

```diff
@@ -1062,3 +1108,30 @@ parts:
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/app-service-configurable/Attribution.txt"
       install -DT "./LICENSE" \
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/app-service-configurable/LICENSE"
+
+  kuiper:
+    source: https://github.com/emqx/kuiper.git
+    source-tag: 0.3.2
+    plugin: make
+    after: [go113]
+    override-build: |
+      export PATH="$SNAPCRAFT_STAGE/go1.13/bin:$GOPATH/bin:$PATH"
+
+      cd $SNAPCRAFT_PART_SRC
+      export BUILD_PATH=$SNAPCRAFT_PART_BUILD
+      export PACKAGES_PATH=$SNAPCRAFT_PART_INSTALL
+      make build_with_edgex
+      make real_pkg
+      cd $SNAPCRAFT_PART_INSTALL
+      tar -xvf kuiper*.tar.gz --strip-components=1
+      rm *.zip *.gz
+
+      install -DT "$SNAPCRAFT_PART_SRC/LICENSE" \
+         "$SNAPCRAFT_PART_INSTALL/usr/share/doc/kuiper/LICENSE"
+    organize:
+      bin/cli: bin/kuiper-cli
+      bin/server: bin/kuiper-server
+    stage:
+      - -etc/mqtt_source.yaml
+    stage-packages:
+      - libzmq3-dev
```

Comment:
this is already staged by the edgex-go part. snapcraft is smart enough to not care since the file contents will be the same AFAIK, but just noticing this

Reply:
also relatedly, why are we staging the dev package here, when we also stage `libzmq5` in app-service-config ? something smells wrong